src/Uno.UI.Tests/ResourceLoader/Strings/en/Resources.Designer.cs

```diff
@@ -61,29 +61,29 @@ internal class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to App70-en.
+        ///   Looks up a localized string similar to .
```

Comment:
Similar to ?

Reply:
Those are generated files and the ordering has changed. We can ignore this.